src/nbt/tag.rs

```diff
@@ -25,10 +25,13 @@ pub enum NbtTag {
 }
 
 impl NbtTag {
+    /// Returns the id associated with the type of data this instance holds
```

Comment:
I don't like `this instance holds` part...

Reply:
I didn't like it too, but I wasn't able to come up with anything better.
I'll just change it to 
/// Returns the id associated with the data type